dev/com.ibm.ws.security.acme_fat/fat/src/com/ibm/ws/security/acme/docker/pebble/PebbleContainer.java

```diff
@@ -120,6 +123,16 @@ public void stop() {
 		super.stop();
 		network.close();
 	}
+	
+	public void stopChallengeServer() {
```

Comment:
Not sure about this. Our goal was to have the CAContainer's be identical. Why are we stopping the challenge server? If we are doing it to stop the mock DNS server, then we should name it "stopDnsServer()" or something similar. The CAContainer should have a method that throws UnsupportedOperationException for any CAContainer impls that can't stop the DNS.

See getOcspResponderUrl().

Reply:
I added this so I could test the Scheduler hitting a failing condition. I tried restarting the Pebble Container, but it could come up with a different Directory URI port. Restarting the challenge server allowed the scheduler to fail on a renew request and then succeed after the challenge server was restarted. But I will update the name and imp.